lib/analytics_data.rb

```diff
@@ -0,0 +1,55 @@
+class AnalyticsData
+  def initialize(elasticsearch_url, indices)
+    @elasticsearch_url = elasticsearch_url
+    @indices = indices
+  end
+
+  def headers
+    columns.keys
+  end
+
+  def rows
+    client = Elasticsearch::Client.new(host: @elasticsearch_url)
+
+    query = {
+      "query": {
+        "match_all": {}
+      }
+    }
+
+    ScrollEnumerator.new(client: client, index_names: @indices, search_body: query) do |hit|
+      item = hit["_source"]
+
+      columns.values.map { |i| i.call(item) }
```

Comment:
This will be slower than explicit declarations, have we checked it not a bit slowdown (don't believe it would be)